src/index.js

```diff
@@ -1,40 +1,64 @@
-'use strict';
+"use strict";
 
-const $tagsInput = document.getElementById('tags-input');
-const $tagsList = document.getElementById('tags-list');
-const tagsArr = ['1', 'super tag'];
+const DEFAULT_TAGS = [
+  {
+    text: "tag1",
+    key: Date.now() + 1
```

Comment:
why you created key via "Date.now() + 1" 
you can simplify it just put "1" and "2"
👎